pkg/alertmanager/api.go

```diff
@@ -517,3 +524,16 @@ func validatePagerDutyConfig(cfg config.PagerdutyConfig) error {
 
 	return nil
 }
+
+// validatePushoverConfig validates the Pushover config and returns an error if it contains
+// settings not allowed by Mimir.
+func validatePushoverConfig(cfg config.PushoverConfig) error {
```

Comment:
Is `Pushover` something new? Should this be mentioned in the changelog?

Reply:
It's not something new. Alertmanager just added 2 new config options, we don't support (because read from file). So not difference for the user.